src/IBM.WatsonDeveloperCloud.VisualRecognition.v3/VisualRecognitionService.cs

```diff
@@ -30,7 +30,7 @@ namespace IBM.WatsonDeveloperCloud.VisualRecognition.v3
 {
     public partial class VisualRecognitionService : WatsonService, IVisualRecognitionService
     {
-        const string SERVICE_NAME = "visual_recognition";
```

Comment:
What's the reasoning for this and the Assistant name change?

Reply:
So we had this `SERVICE_NAME` generated before using `apiServiceName` but we didn't really use it for anything until we had the ibm-credentials.env file. When I originally implemented it I had the name wrong in the credentials file I was using which was only uncovered after doing the unity work that correctly grabs it from `vcapServicesName`. After I make the change to the generator it should generate `SERVICE_NAME` correctly.

Reply:
So you're saying that for the release, these will be `assistant` and `visual_recognition`, right? Just verifying because those are the values that will be in the credential file.

Reply:
ah ok, i'm confused then. I thought these were supposed to be vcapServicesName. It was right the first time then. I'll switch these back and revise the Unity generator to get the correct service name.

Reply:
I updated these back to the correct names `assistant` and `visual_recognition`. Thanks for straightening that out!

Reply:
No worries. Sorry for the confusion!